docs/backends.rst

```diff
@@ -89,15 +89,15 @@ make sure the data model is correctly created::
 
 Please make sure to apply the database migrations::
 
-    python manage.py migrate database
+    python manage.py migrate constance_db
```

Comment:
Won't this break the migrations system since you're basically moving the app? If yes, then this is a no-go since it'd break every deployed version of constance hard and would require faking the constance migrations again. Not worth the hassle.